src/Pages/Authentication/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,132 @@
+          }
+        })
+        .catch((response) => {
+          if (response.data && response.data.error) {
+            setAlertContent({_id: uuid(), isShow:true, type:'ERROR', content:"Unexpected error.Login Failed"})
+          }
+        })
+    }
+        
+
+    return(
+        <>
+            <section className="auth">
+                <div className="col-7 auth-img-container">
+                    <img src={blitheCollage}
+                        alt="auth-img" />
+                </div>
+                <div className="col-7 auth-content-container">
+                    <div className="auth-form">
+                        <img src={blitheLogo} className="blithe-logo" alt="blithe-logo" />
+                        <form>
+                            <p className="heading-md heading">Sign Up</p>
+                            <InputField 
+                                type='email'
+                                title='Email'
+                                name='email'
+                                placeholder='Enter email'
+                                value={sigUpData?.email? sigUpData.email : ''}
+                                onChange={(e)=>setSignUpData(prevData => {return  {...prevData, email:e.target.value}})}
+                            />
+                            <InputField 
+                                type='text'
+                                title='FirstName'
+                                name='firstName'
+                                placeholder='Enter firstname'
+                                value={sigUpData?.firstName? sigUpData.firstName : ''}
+                                onChange={(e)=>setSignUpData(prevData => {return  {...prevData, firstName:e.target.value}})}
+                            />
+                            <InputField 
+                                type='text'
+                                title='LastName'
+                                name='lastName'
+                                placeholder='Enter lastname'
+                                value={sigUpData?.lastName? sigUpData.lastName : ''}
+                                onChange={(e)=>setSignUpData(prevData => {return  {...prevData, lastName:e.target.value}})}
+                            />
+                            <InputField 
+                                type={showPassword?'text':'password'}
+                                title='Password'
+                                name='password'
+                                placeholder='Enter password'
+                                showPassword={showPassword}
+                                handlePassword = {handlePassword}
+                                value={sigUpData?.password? sigUpData.password : ''}
+                                onChange={(e)=>setSignUpData(prevData => {return  {...prevData, password:e.target.value}})}
```

Comment:
The return part wont need it here  i believe, you can do the same like 
onChange{(e) => setSignUpData( prevData=> {...prevData, password:e.target.value}  )  }

Reply:
its showing error when not using return here !